src/Game.cpp

```diff
@@ -826,12 +826,11 @@ void Game::LoadLandscape(const std::filesystem::path& path)
 	{
 		throw std::runtime_error("Could not find landscape " + path.generic_string());
 	}
-
-	_landIsland = std::make_unique<LandIsland>();
-	_landIsland->LoadFromFile(fixedName);
+	Locator::terrainSystem::emplace<LandIsland>();
```

Comment:
This should be done in initialize.

Reply:
I noticed some inconsistency in our code, our locator inits are in `Game::Initialize()`, `Locator::InitializeLevel()` or `InitializeGame()`

Reply:
I put the terrainSystem initilizer inside the Locator::InitializeLevel(), it seemed like the most appropriate place!